app/src/main/java/org/mozilla/focus/home/HomeFragment.java

```diff
@@ -966,16 +1055,25 @@ private static void doWithActivity(Activity activity, DoWithThemeManager doWithT
         }
     }
 
+
     private class GestureListenerAdapter implements OnSwipeListener {
 
         @Override
         public void onSwipeUp() {
-            btnMenu.performClick();
+            if (contentPanel != null) {
+                contentPanel.show(true);
```

Comment:
maybe also set the priority to "User" here. Since the user had already seen the news, we shouldn't change it after this event. However, this will add disk r/w every time the user swipes up.... which may be fine.

Reply:
the user had seen the news. Treat it as an user selection so no on can change it